docs/proposals/local-cluster-ux.md

```diff
@@ -0,0 +1,184 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Kubernetes Local Cluster Experience
+
+This proposal attempts to improve the existing local cluster experience for kubernetes.
+The current local cluster experience is sub-par and often not functional.
```

Comment:
Some of the deployment solutions just don't work. It should work all the
time.
I assume you were only thinking of the docker based solution?

On Mon, Apr 18, 2016 at 12:02 PM, Lucas Käldström <notifications@github.com>
wrote:

> In docs/proposals/local-cluster-ux.md
> <https://github.com/kubernetes/kubernetes/pull/24106#discussion_r60115283>
> :
>
> > +If you are using a released version of Kubernetes, you should
> > +refer to the docs that go with that version.
> > +
> > +Documentation for other releases can be found at
> > +[releases.k8s.io](http://releases.k8s.io).
> > +</strong>
> > +--
> > +
> > +<!-- END STRIP_FOR_RELEASE -->
> > +
> > +<!-- END MUNGE: UNVERSIONED_WARNING -->
> > +
> > +# Kubernetes Local Cluster Experience
> > +
> > +This proposal attempts to improve the existing local cluster experience for kubernetes.
> > +The current local cluster experience is sub-par and often not functional.
>
> Yes, it has some shortcomings, but I wouldn't say often not functional
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24106/files/4d4288e76854d11c1f2db2ffd002a3ed041995fe#r60115283>
>